feat: :sparkles: IPFS Readme (#1)
ipfs/.github: profile/README.md

<p align="center">

>   <img src="https://github.com/ipfs/.github/blob/main/profile/ipfs-powered.png" width="100%" alt="IPFS Powered">

Reviewer: We should be able to use a relative link here, right? This would make MD previews work as expected.

```suggestion
  <img src="ipfs-powered.png" width="100%" alt="IPFS Powered">
```

Author: since this is a special repo, it doesn't translate relative links, I tried!

- AppWrite: https://github.com/appwrite/.github/blob/main/profile/README.md?plain=1#L3
- MS: https://github.com/microsoft/.github/blob/main/profile/README.md?plain=1#L1

Reviewer: Thanks for checking :)

</p>

The InterPlanetary File System (IPFS) is a set of composable, peer-to-peer protocols for addressing, routing, and transferring [content-addressed data](https://docs.ipfs.tech/concepts/glossary/#content-addressing) in a decentralized file system. Many popular Web3 projects are built on IPFS - see the [ecosystem directory](https://ecosystem.ipfs.tech/) for a list of some of these projects.

> Exceptionally talented open communities are building the future of the decentralized web. There are excellent "good first issues" that need your attention and support! Join the movement, get involved today!

Reviewer: Do we want to link to `good first issue` only or `help wanted` too? I included both but in the search link suggested here but maybe `good first issue` would be a better curated starting point?

```suggestion
Exceptionally talented open communities are building the future of the decentralized web. Excellent [good first issues](https://github.com/search?q=org%3Aipfs+label%3A%22good+first+issue%22%2C%22help+wanted%22&type=issues) need your attention and support! Join the movement! Get involved today!
```

Author: Nice, I quickly tried, but couldn't search through using labels. this lgtm!

### Documentation

- [IPFS Docs](https://docs.ipfs.tech/)
- [IPFS Specs](https://specs.ipfs.tech/)
- [Getting Started](https://ipfs.tech/#install)

> - [How IPFS Works](https://ipfs.tech/#how)

Reviewer 2: Could we change the order, so it is a progressive on ramp? 
Most users will not need to read specs.

```suggestion
- [How IPFS Works](https://ipfs.tech/#how)
- [Getting Started](https://ipfs.tech/#install)
- [Documentation Portal](https://docs.ipfs.tech/)
- [Specs for Implementers](https://specs.ipfs.tech/)
```

### Implementations

> - [kubo (go)](https://github.com/ipfs/kubo)
> - [helia (JS)](https://github.com/ipfs/helia)

Reviewer: I'm not sure whether we want Helia capitalised too. I can see both versions in its' README.

```suggestion
- [Kubo (Go)](https://github.com/ipfs/kubo)
- [Helia (JS)](https://github.com/ipfs/helia)
```
